tests/test.py

```diff
@@ -257,60 +86,6 @@ def test_backend_exception_is_wrapped(self):
         with self.assertRaises(BackendError):
             self.device.get_hardware_version()
 
-    def test_i3c_successful_write_read_operations_on_target(self):
-        if not self.use_simulator:
-            self.skipTest("For simulator only")
-
-        i3c = self.device.create_interface("i3c.controller")
-
-        i3c.init_bus(3300)
-
-        subaddress = [0x00, 0x00]
-
-        (success, result) = i3c.write(
-            0x08,
-            i3c.TransferMode.I3C_SDR,
-            subaddress,
-            [0xDE, 0xAD, 0xBE, 0xEF]
-        )
-
-        (success, result) = i3c.read(
-            0x08,
-            i3c.TransferMode.I3C_SDR,
-            subaddress,
-            4
-        )
-
-        self.assertTupleEqual((success, result), (True, [0xDE, 0xAD, 0xBE, 0xEF]))
-
-    def test_ccc_getpid(self):
-        if not self.use_simulator:
-            self.skipTest("For simulator only")
-
-        i3c = self.device.create_interface("i3c.controller")
-
-        i3c.init_bus(3300)
-
-        (success, result) = i3c.ccc_getpid(0x08)
-
-        self.assertTupleEqual((success, result), (True, [0x00, 0x00, 0x00, 0x00, 0x64, 0x65]))
-
-    def test_ccc_rstdaa(self):
-        if self.use_simulator:
-            self.skipTest("For real device only")
-
-        i3c = self.device.create_interface("i3c.controller")
-
-        i3c.init_bus(3300)
-
-        (success, result) = i3c.ccc_rstdaa()
-
-        self.assertTupleEqual((success, result), (True, None))
-
-        (success, result) = i3c.ccc_getpid(0x08)
-
-        self.assertTupleEqual((success, result), (False, "NACK_ERROR"))
-
```

Comment:
I like you moved the I3C related tests to i3c_tests.py and i3c_ccc_tests.py. We should do the same in the future for SPI so we have in test.py only "system" related tests, do we have a ticket for this?